### DISCUSSION TITLE
Is it possible to 'remove' the installed scanner module for testing purposes

### DISCUSSION THREAD

**Discussion by Toverbal** [capawesome-team/capacitor-mlkit#86] [Q&A] [answered]
Hi, I'm using the isGoogleBarcodeScannerModuleAvailable and installGoogleBarcodeScannerModule methods (with the callback) and they seem to work fine. But it works only once and so, once installed, the isGoogleBarcodeScannerModuleAvailable is true. Great for users, but not for me as I want to tweak the progress callback. So, long story short: is it possible to somehow 'remove' the installed scanner again, so I can test it more than once?

Thanks a lot for the answer in advance!

**Comment by robingenz** [comment, marked as answer]
It seems that there is a way, see https://developers.google.com/android/guides/module-install-apis?hl=en#local_testing_with_fakemoduleinstallclient.
However, this is currently not yet supported by the plugin. Feel free to create a feature request. PRs are welcome.
